src/main/java/book/project/bookstore/service/BookServiceImpl.java

```diff
@@ -36,4 +37,24 @@ public BookDto findById(Long id) {
         );
         return bookMapper.toDto(book);
     }
+
+    @Override
+    public BookDto update(Long id, UpdateBookRequestDto bookRequestDto) {
+        Book book = bookRepository.findById(id).map(existingBook -> {
+            existingBook.setTitle(bookRequestDto.getTitle());
+            existingBook.setDescription(bookRequestDto.getDescription());
+            existingBook.setAuthor(bookRequestDto.getAuthor());
+            existingBook.setPrice(bookRequestDto.getPrice());
+            existingBook.setIsbn(bookRequestDto.getIsbn());
+            existingBook.setCoverImage(bookRequestDto.getCoverImage());
```

Comment:
use Mapper for it:
Book updateBook(Book book, UpdateBookRequestDto bookRequestDto);